src/fheroes2/dialog/dialog_levelup.cpp

```diff
@@ -77,9 +77,7 @@ int DialogOneSecondary( const Heroes & hero, const std::string & name, const int
 
 int DialogSelectSecondary( const std::string & name, const int primarySkillType, const Skill::Secondary & sec1, const Skill::Secondary & sec2, Heroes & hero )
 {
-    std::string header = _( "%{name} has gained a level." );
-    header.append( "\n \n" );
-    header.append( _( "%{skill} +1" ) );
+    std::string header = _( "%{name} has gained a level.\n%{skill} +1" );
```

Comment:
Was the empty text string removed for some reason? :)
```suggestion
    std::string header = _( "%{name} has gained a level.\n\n%{skill} +1" );
```

Reply:
Nope. I think my rendering was off with the old text. I fixed it.